File: react-native/src/theta-repository/digest-auth.ts
/**
 * Authentication information used for client mode
 */
export type DigestAuth = {
  /**
   * User name
   */
  username: string;
  /**
   * Password
   *
   * If omitted, the default password is used.
   * Default password is "THETA" + "XX" after the beginning of the serial number.
   */
  password?: string;
};

File: flutter/lib/digest_auth.dart
/// Authentication information used for client mode
class DigestAuth {
  /// User name
  String username;

  /// Password
  ///
  /// If omitted, the default password is used.
  /// Default password is "THETA" + "XX" after the beginning of the serial number.
  String? password;

  DigestAuth(this.username, [this.password]);
}
